bigchaindb/pipelines/block.py

```diff
@@ -73,10 +74,15 @@ def validate_tx(self, tx):
 
         # If transaction is not valid it should not be included
         try:
+            if tx.operation == Transaction.GENESIS:
+                blocks_count = backend.query.count_blocks(self.bigchain.connection)
```

Comment:
I don't think there is a need for this check. When the pipelines are started they expect that bigchaindb was already initialized or else it will fail.